tests/playbooks/conjurize-container.yml

```diff
@@ -8,5 +8,5 @@
       conjur_host_factory_token: "{{lookup('env', 'HFTOKEN')}}"
       conjur_ssl_certificate: "{{lookup('file', '../conjur.pem')}}"
       conjur_host_name: "conjur_{{ ansible_hostname }}"
-      conjur_validate_certs: no
+      conjur_validate_certs: yes
```

Comment:
Do we really need this boolean? Can't we use a "when something is defined"? (maybe 'conjur_ssl_certificate')